api/app/controllers/mno_enterprise/jpi/v1/admin/invoices_controller.rb

```diff
@@ -13,11 +13,18 @@ def show
 
     # GET /mnoe/jpi/v1/admin/invoices/current_billing_amount
     def current_billing_amount
-      data = Rails.cache.fetch('tenant_admin/current_billing_amount', expires_in: ADMIN_CACHE_DURATION) do
-        billing = MnoEnterprise::Organization.all.map(&:current_billing).sum(Money.new(0))
-        {current_billing_amount: {amount: billing.amount, currency: billing.currency_as_string}}
-      end
-      render json: data
+      tenant = MnoEnterprise::Tenant.get('tenant')
```

Comment:
To get a singular resource, do we need to specify a fake id? Could it be `MnoEnterprise::Tenant.get` ?